src/channel.js

```diff
@@ -0,0 +1 @@
+module.exports = '__THEMING__';
```

Comment:
i understand that its in global namespace, but the issue is a bit exagerated. it worked pretty fine for styled-components. and it works consistently. with random channel, its hard to debug  to begin with and then it works inconsistently and on top of that you still can get conflicts